aten/src/ATen/Parallel.h

```diff
@@ -51,4 +51,35 @@ void parallel_for(
   }
 }
 
+template <class scalar_t, class F, class SF>
+inline scalar_t parallel_reduce(
+    int64_t begin,
+    int64_t end,
+    int64_t grain_size,
+    scalar_t ident,
+    F f,
+    SF sf) {
```

Comment:
Any chance for a better name than sf? It’s not clear at all what is that supposed to be